gostrftime.go

```diff
@@ -110,7 +110,8 @@ func strftime(b *bytes.Buffer, c rune, t time.Time) error {
 		b.WriteString(zone)
 	case 'z':
 		_, offset := t.Zone()
-		fmt.Fprintf(b, "%+05d", offset)
+		allMinutes := int(offset/60)
+		fmt.Fprintf(b, "%+02d%02d", int(allMinutes / 60), int(allMinutes % 60))
```

Comment:
Should be `"%+03d%02d"` so that the + is followed by two digits. As it is now you can end up with numbers like `+159`.
Other than that, this looks ok. Thanks!